hugolib/page.go

```diff
@@ -1021,6 +1021,14 @@ func (p *Page) update(f interface{}) error {
 			p.Date = fi.ModTime()
 			p.Params["date"] = p.Date
 		}
+	} else if p.Date.IsZero() && p.s.Cfg.GetBool("useFilenameDateAsFallback") {
```

Comment:
I'm not sure I follow the logic here. Are you saying we could reduce the number of configs to 1? How?